src/service/vsync-impl.js

```diff
@@ -375,12 +375,22 @@ export class Vsync {
     this.states_ = this.nextStates_;
     for (let i = 0; i < tasks.length; i++) {
       if (tasks[i].measure) {
-        tasks[i].measure(states[i]);
+        try {
```

Comment:
Consider making a `*NoInline` function to avoid the try block optimizing the function. Not sure it matters here.